### PR TITLE
Pull request for adding navbar and footer into dev branch

### PR DESCRIPTION
**Pull request by nash4comp**
Hello, please review my update and accept my request if there's no issue.

Details are below.
- Added navbar and footer HTML
- Added includeHTML.js for inserting navbar.html and footer.html
- Added logo image and icon for favicon
- Put some code for navbar and footer into index.html

Media query will be updated later.

Thanks.

### CODE REVIEW

File: app/html/index.html [hunk 1]
```diff
@@ -16,7 +34,11 @@
     <p id="loginStatus">&nbsp;</p>
   </div>
 
-  <script src="js/index.js"></script>
```
Comment (mksample) [inline app/html/index.html:19]: This script should be added back in, it handles login/logout.

File: app/html/index.html [hunk 2]
```diff
@@ -16,7 +34,11 @@
...
+  <!-- footer.html will be in here -->
+  <footer include-html="/html/footer.html"></footer>
 </body>
+<script>
```
Comment (mksample) [inline app/html/index.html:40]: I'd include this script in the body similar to how the other one was.
Example: 
```
<body>

<script>
    includeHTML();
</script>
</body>
```

File: app/html/test.html [hunk 3]
```diff
@@ -0,0 +1,38 @@
+<!-- This file is only for the test of Navbar and Footer.
```
Comment (mksample) [inline app/html/test.html:1]: What is the purpose of this file?

**Review by mksample — COMMENTED** [review]
Left a couple of comments. Simple stuff about the script in index.html and the test.html file.

File: app/html/test.html (hunk 3 above)
Comment (nash4comp) [reply app/html/test.html:1]: It's just locally test file. So it should be deleted. I missed it to be deleted.

File: app/html/index.html (hunk 1 above)
Comment (nash4comp) [reply app/html/index.html:19]: In fact, I didn't deleted it, but just moved it to the top side. I meant to bring the importing the code for javascript together, but I think it's right to put it back there if you need it here.

File: app/html/index.html (hunk 2 above)
Comment (nash4comp) [reply app/html/index.html:40]: Okay!

File: app/html/index.html (hunk 1 above)
Comment (mksample) [reply app/html/index.html:19]: ahhhh ok I missed that, disregard this then :)

**Review by mksample — APPROVED** [review]
LGTM!

**Pull request merged**